lib/client/mfa/cli.go

```diff
@@ -204,6 +197,8 @@ func (c *CLIPrompt) Run(ctx context.Context, chal *proto.MFAAuthenticateChalleng
 	case promptSSO:
 		resp, err := c.promptSSO(ctx, chal)
 		return resp, trace.Wrap(err)
+	case promptOTP && useSessionMFA:
+		return nil, trace.AccessDenied("only WebAuthn and SSO MFA methods are supported with per-session MFA, please register a supported MFA method to connect to this server")
```

Comment:
What do we mean by "server" here? Isn't `CLIPrompt.Run` used for all kinds of things that need MFA?

I think we could drop "to connect to this server".